src/ui/screens/AddAccountScreen/AddAccountScreen.tsx

```diff
@@ -112,7 +118,7 @@ export function AddAccountScreen({navigation}: {navigation: NavigationProp<AppSt
                       </TabScreen>
                       <TabScreen label="Via QR" icon="qrcode">
                         <View style={globalStyles.paddedContainer}>
-                          <QRCamera onRead={handleScan} />
+                          {tabIndex === 1 && <QRCamera onRead={handleScan} />}
```

Comment:
This is to block the rendering of `QRCamera` component until navigated to that tab. Otherwise it scans any visible QR even if not navigated to the tab.